chapter_recurrent-neural-networks/sequence.md

```diff
@@ -111,6 +124,38 @@ def get_net():
 loss = gluon.loss.L2Loss()
 ```
 
+```{.python .input}
+#@tab pytorch
+tau = 4
+features = torch.zeros((T-tau, tau))
+for i in range(tau):
+    features[:, i] = x[i: T-tau+i]
+labels = x[tau:]
+
+batch_size, n_train = 16, 600
+train_iter = d2l.load_array((features[:n_train], labels[:n_train]),
+                            batch_size, is_train=True)
+test_iter = d2l.load_array((features[:n_train], labels[:n_train]),
+                           batch_size, is_train=False)
+
+# Function for initializing the weights of net
+def init_weights(m):
+    if type(m) == nn.Linear:
+        torch.nn.init.xavier_uniform_(m.weight)
+
+# Vanilla MLP architecture
+def get_net():
+    net = nn.Sequential()
```

Comment:
we didn't introduce `add_module` before, we can use `nn.Sequential(nn.xxx)` as much as possible

Reply:
Yes, I can go do that. I thought `add_module` is a bit more clean implementation wise. But I'll send in changes if we want to stick with `nn.Sequential(nn.xx , nn.xx , ... , nn.xx )`